src/main/java/lazyprogrammer/jwtdemo/entities/Roles.java

```diff
@@ -0,0 +1,17 @@
+package lazyprogrammer.jwtdemo.entities;
+
+public enum Roles {
```

Comment:
We could use database to manage the roles properly. Because using ENUM will make it more tightly coupled and less extensible.